app/src/main/java/com/ciandt/dragonfly/example/data/mapper/ServicesToComparisonResult.kt

```diff
@@ -0,0 +1,27 @@
+package com.ciandt.dragonfly.example.data.mapper
+
+import com.ciandt.dragonfly.example.data.remote.entities.Service
+import com.ciandt.dragonfly.example.features.feedback.model.ComparisonResult
+import com.ciandt.dragonfly.tensorflow.Classifier
+
+class ServicesToComparisonResult(val services: List<Service>) : Mapper<ComparisonResult>() {
```

Comment:
BenchmarksToComparisonResult? (not comfortable with this ComparisonResult term, but can't come with a better option).